### PR TITLE
Cohortget

### PR DESCRIPTION
**Pull request by HankCaswell**
Added a GET method to return a list of all cohorts, updated documentation with parameters for the GET request.

### CODE REVIEW

File: backend/API-DOCUMENTATION.md [hunk 1]
```diff
@@ -83,6 +83,22 @@ You have requested a password reset. Please click the link below to reset your p
 - **Response** :
     - Success: HTTP 204
 
+
+## GET A LIST OF COHORTS
+- NOTE: There should be only 1 record
```
Comment (DeeJaeMann) [inline backend/API-DOCUMENTATION.md:88]: Is this note right?

**Review by DeeJaeMann — COMMENTED** [review]
Just a comment in the API Documentation section.  I'm not sure if this means there's a hard limit to cohorts or if currently we're just dealing with 1 cohort in the db

File: backend/API-DOCUMENTATION.md (hunk 1 above)
Comment (HankCaswell) [reply backend/API-DOCUMENTATION.md:88]: Removed.

**Review by DeeJaeMann — APPROVED** [review]
Good to go

**Pull request merged**